doc/classes/EditorSettings.xml

```diff
@@ -1,5 +1,6 @@
 <?xml version="1.0" encoding="UTF-8" ?>
-<class name="EditorSettings" inherits="Resource" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="../class.xsd">
+<class name="EditorSettings" inherits="Resource"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="../class.xsd">
```

Comment:
This should be restored, check the CI check it corrects this

Reply:
Sorry for the total beginner questions. It keeps auto formatting like this on save. Is this my editor, or a project configured formatter doing this?

Reply:
It must be your editor enforcing a line limit, it should be just one line

Reply:
Fixed